src/Wallabag/CoreBundle/Resources/views/themes/baggy/Entry/entry.html.twig

```diff
@@ -54,7 +54,6 @@
                 {% endif %}
             </i>
 
-            {% set nbAnnotations = entry.annotations | length %}
```

Comment:
I tested both themes and this line was not helping in the baggy theme either – both themes showed a placeholder instead of the number of annotations.